src/clover/optimisation/appraisal.py

```diff
@@ -237,7 +243,79 @@ def _simulation_environmental_appraisal(  # pylint: disable=too-many-locals
     )
 
 
+def _get_grid_pricing_tier(
```

Comment:
## 💲 Move to finance?
I think this function should probably sit in the finance module as it's, by nature, financial, or, alternatively, in the grid module, as it's to do with the grid :smile:

Reply:
@BenWinchester Can you check this new update of the get pricing tier, i think this looks decent and should work (I understood the concept behind it). However, can you please let me know how to write down the monthly consumption based on the grid_energy !!?